wagtail/admin/templates/wagtailadmin/shared/dialog/dialog.html

```diff
@@ -29,14 +29,19 @@
             {% endif %}
 
             <div class="w-dialog__content">
-                <h2 class="w-dialog__title w-h1" id="title-{{ id }}">
-                    {% if icon_name %}{% icon name=icon_name classname="w-dialog__icon" %}{% endif %}
-                    {{ title }}
-                </h2>
-
-                {% if subtitle %}
-                    <p class="w-dialog__subtitle w-help-text">{{ subtitle }}</p>
-                {% endif %}
+                <div class="w-dialog__header">
+                    <h2 class="w-dialog__title w-h1" id="title-{{ id }}">
+                        {% if icon_name %}{% icon name=icon_name classname="w-dialog__icon" %}{% endif %}
+                        {{ title }}
+                    </h2>
+                    {% if subtitle %}
+                        <p class="w-dialog__subtitle w-help-text">{{ subtitle }}
+                            {% fragment as issues_found %}
+                                {% trans 'Issues found' %} {% if a11y_count %}<span class="w-a11y-result__count" data-a11y-result-count></span>{% endif %}
+                            {% endfragment %}
```

Comment:
This fragment should be removed now that it's been done in the `userbar/item_accessibility.html` template